lib/dbservice_web/controllers/session_controller.ex

```diff
@@ -210,6 +213,53 @@ defmodule DbserviceWeb.SessionController do
     end
   end
 
+  swagger_path :search do
+    post("/api/session/search")
+    summary("Bulk search sessions by platform IDs")
+    description("Search for multiple sessions by providing a list of platform_ids")
+
+    parameters do
+      body(:body, Schema.ref(:SessionSearch), "Search parameters", required: true)
+    end
+
+    response(200, "OK", Schema.ref(:Sessions))
+  end
+
+  def search(conn, params) do
+    platform_ids = Map.get(params, "platform_ids", [])
+    platform = Map.get(params, "platform")
+
+    query = build_search_query(platform_ids, platform, params)
+    sessions = Repo.all(query)
+
+    render(conn, :index, session: sessions)
+  end
+
+  defp build_search_query(platform_ids, platform, params) do
+    sort_order = extract_sort_order(params)
+
+    base_query =
+      from m in Session,
+        order_by: [{^sort_order, m.id}],
+        offset: ^params["offset"],
+        limit: ^params["limit"]
+
```

Comment:
I think this query is already present in index function. Let's create a common function for this?